tsconfig.json

```diff
@@ -14,7 +14,7 @@
     "target": "es5",                                     /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */
     // "lib": [],                                        /* Specify a set of bundled library declaration files that describe the target runtime environment. */
     // "jsx": "preserve",                                /* Specify what JSX code is generated. */
-    // "experimentalDecorators": true,                   /* Enable experimental support for TC39 stage 2 draft decorators. */
+    // "experimentalDecorators": true,                   /* Enable experimental supports for TC39 stage 2 draft decorators. */
```

Comment:
Was this supposed to change?

Reply:
i think this was coming from when we merged support into master, then i did a rebase  master in the current branch

Reply:
It's a comment, it probably doesnt matter